docker-compose.yml

```diff
@@ -1,53 +1,68 @@
-version: "2"
+version: '3'
 
 services:
 
-  db:
+  mattermost_db:
+    hostname: "db.mattermost.local"
+    container_name: "mattermost_db"
     build: db
     restart: unless-stopped
     volumes:
       - ./volumes/db/var/lib/postgresql/data:/var/lib/postgresql/data
       - /etc/localtime:/etc/localtime:ro
-    environment:
-      - POSTGRES_USER=mmuser
-      - POSTGRES_PASSWORD=mmuser_password
-      - POSTGRES_DB=mattermost
-    # uncomment the following to enable backup
-    #  - AWS_ACCESS_KEY_ID=XXXX
-    #  - AWS_SECRET_ACCESS_KEY=XXXX
-    #  - WALE_S3_PREFIX=s3://BUCKET_NAME/PATH
-    #  - AWS_REGION=us-east-1
+    env_file:
+      - .env
+    networks:
+      mattermost_network:
+        ipv4_address: 172.22.2.4
 
-  app:
+  mattermost_app:
+    hostname: "app.mattermost.local"
+    container_name: "mattermost_app"
     build:
       context: app
       # comment out 2 following lines for team edition
-      # args:
-      #   - edition=team
+      args:
+        - edition=team
     restart: unless-stopped
+    links:
```

Comment:
We removed `links` in PR https://github.com/mattermost/mattermost-docker/pull/185 because it is deprecated. Should remove it I guess.

Reply:
The links is because you can assign one of more aliases for the different hosts; i think that, for the moment, docker-compose will mantain retrocompatibility